resty/src/App.js

```diff
@@ -13,46 +14,47 @@ class App extends React.Component {
       count: '',
       resultsArray: [],
       headers: {},
-      history: {},
-      storage: {}
     }
   }
 
   updateState = (data) => {
-    this.setState({ resultsArray: data.results })
+    // console.log('get from fake api??', data);
+    this.setState({ resultsArray: data })
     this.setState({ count: data.count })
   }
   updateHeaders = (headers) => {
     this.setState({ headers })
   }
-  updateStorage = (storage) => {
-    console.log('res???', storage)
-    this.setState({ storage })
+  updateQueries = (queries) => {
+    // console.log('res???', storage)
+    this.setState({ queries })
   }
 
   render() {
     return (
-      <>
+      <BrowserRouter>
         <Header />
 
         <Form
           update={this.updateState}
           headers={this.updateHeaders}
-          query={this.updateStorage}
+          query={this.updateQueries}
         />
 
         <Results
           count={this.state.count}
           results={this.state.resultsArray}
           headers={this.state.headers}
         />
-        {/* 
+
         <History
-          history={this.state.history}
-        /> */}
+          queryHistory={this.state.queries}
+        />
 
         <Footer></Footer>
-      </>
+
+      </BrowserRouter>
+
     )
   }
 }
```

Comment:
Would like to see that history working so that I can see all the api's that I have looked at before
I would like to see the help page link working as well